examples/controller_pid.py

```diff
@@ -0,0 +1,226 @@
+import argparse
+
+import fym
+import matplotlib.pyplot as plt
+import numpy as np
+
+import ftc
+from ftc.models.multicopter import Multicopter
+from ftc.utils import safeupdate
+
+np.seterr(all="raise")
+
+
+class MyEnv(fym.BaseEnv):
+    ENV_CONFIG = {
+        "fkw": {
+            "dt": 0.01,
+            "max_t": 20,
+        },
+        "plant": {
+            "init": {
+                "pos": np.vstack((0.5, 0.5, 0.0)),
+                "vel": np.zeros((3, 1)),
+                "quat": np.vstack((1, 0, 0, 0)),
+                "omega": np.zeros((3, 1)),
+            },
+        },
+    }
+
+    def __init__(self, env_config={}):
+        env_config = safeupdate(self.ENV_CONFIG, env_config)
+        super().__init__(**env_config["fkw"])
+        self.plant = Multicopter(env_config["plant"])
```

Comment:
```suggestion
        self.plant = Multicopter(env_config["plant"], rtype="quad")
```
로 해야 쿼드콥터에 대한 시뮬레이션이 돼요!